qngng/qngng.py

```diff
@@ -272,7 +272,9 @@ class _Format(enum.Enum):
 
 def _format_name(fullname, fmt=_Format.DEFAULT, with_middle_name_initial=True):
 
-    if fullname.middle_name is None:
+    if fullname.surname is None:
+        raw_name = '{}'.format(fullname.name)
+    elif fullname.middle_name is None:
```

Comment:
This is becoming hard to read.

Can we have parts instead?

1. Start with an empty list named `parts`.
2. If there's a name, append it to the list.
3. If there's a middle name/initial, append it to the list (using the very sophisticated algorithm which is already there).
4. If there's a surname, append it to the list.
5. Let `raw_name` be `' '.join(parts)`.